### DISCUSSION TITLE
https://blog.eledah.ir/about

### DISCUSSION THREAD

**Discussion by giscus** [eledah/quartz_blog#32] [Announcements]
# https://blog.eledah.ir/about

من کی‌ام؟ سلام 👋 من محمدعلی هستم. ۲۷ ساله از تهران. برای ارتباط، تلگرام پیام بدین و برای آشنایی، صفحهٔ بریده‌ها و ادامهٔ همین متن رو ببینید. علوم و نظریه‌های مورد علاقهٔ من نظریه‌ها لنزهایی هستند که در دیدن واقعیت به ما کمک می‌کنند.

https://blog.eledah.ir/about

**Comment by masoudnasrollahi** [comment]
با تک‌تک پروژه‌هات حال کردم و مخم سوت کشید! حتما باید بیشتر دربارۀ نظریه‌ها بخونم و مشتاقم بیشتر درباره‌شون بدونم.

دربارۀ روانکاوی می‌تونی [پادکست ذهن‌های ناهشیار](https://t.me/MindsOfTheUnconscious) را گوش کنی تا احتمالا جای خودش را بیشتر توی ذهنت باز کنه و تو هم جایگاهش را بهتر متوجه بشی.
پروژۀ مورد علاقۀ من هم که مشخصه، کریستالین! من با این پروژه باهات آشنا شدم و از این آشنایی خوشوقتم😁
پروژهٔ خط زمانی را برای خودم تصور می‌کردم، تا آدم‌هایی که می‌شناسم و ازشون کتابی خونده‌ام را روی محور زمان نشان بدم و هم‌دوره‌ای بودنشون یا تاثیرپذیرفتنشون از همدیگه یا ترتیب حضورشون روی زمین را یکجا نشان بدم. با دیدن پروژۀ تو و ایدۀ پشتش جا خوردم!

**Comment by khezri7577** [comment]
استاد کی پادکستتو راه میندازی؟

**Reply by eledah** [reply]
توی کانال منتشر می‌شه
